### PR TITLE
Release 0.14.9

### PR DESCRIPTION
**Pull request by joshua-kim**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ## Changelog
+### 0.14.9
+* [#370](https://github.com/awslabs/amazon-kinesis-producer/pull/370) Upgraded build script dependencies.
```
Comment (yatins47) [inline CHANGELOG.md:3]: Can we add a little more details here

File: java/amazon-kinesis-producer-sample/pom.xml [hunk 2]
```diff
@@ -56,7 +56,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>amazon-kinesis-producer</artifactId>
-            <version>0.14.8</version>
+            <version>0.14.9</version>
```
Comment (yatins47) [inline java/amazon-kinesis-producer-sample/pom.xml:59]: Wasn't there another place we need to make this change except the pom files ?

Comment (joshua-kim) [reply java/amazon-kinesis-producer-sample/pom.xml:59]: We need to update the cpp core, sop was outdated... added in next revision.

File: CHANGELOG.md (hunk 1 above)
Comment (joshua-kim) [reply CHANGELOG.md:3]: added

**Pull request closed without merging**